ports/glutin/window.rs

```diff
@@ -605,15 +607,21 @@ impl WindowMethods for Window {
     }
 
     fn set_page_title(&self, title: Option<String>) {
+        let fallback_title = match self.current_url.borrow_mut().clone() {
```

Comment:
Honestly though, I'm not really a huge fan of cloning, so perhaps the current_url field can instead store a `RefCell<Option<Url>>` instead.